scout/parse/variant/transcript.py

```diff
@@ -312,3 +202,162 @@ def parse_transcripts_spliceai(transcript, entry):
     transcript["spliceai_delta_score"] = spliceai_delta_score
     transcript["spliceai_delta_position"] = spliceai_delta_position
     transcript["spliceai_prediction"] = spliceai_prediction
+
+
+def get_strand(transcript, entry):
+    """Get string from transcript"""
+    if entry.get("STRAND") == "1":
+        return "+"
+    if entry.get("STRAND") == "-1":
+        return "-"
+    else:
```

Comment:
Redundant else

Reply:
Thanks, will remove `else` but keep `return None` (I know that the return would be redundant as well, but somewhere you've got to draw the line).

Reply:
Done!